tck/features/CreateAcceptance.feature

```diff
@@ -0,0 +1,607 @@
+#
+# Copyright 2016 "Neo Technology",
+# Network Engine for Objects in Lund AB (http://neotechnology.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Feature: CreateAcceptance
+
+  Scenario: Create a single node
+    Given any graph
+    When executing query:
+      """
+      CREATE ()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 1 |
+
+  Scenario: Create a single node with a single label
+    Given any graph
+    When executing query:
+      """
+      CREATE (:A)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 1 |
+      | +labels | 1 |
+
+  Scenario: Create a single node with multiple labels
+    Given any graph
+    When executing query:
+      """
+      CREATE (:A:B:C:D)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 1 |
+      | +labels | 4 |
+
+  Scenario: Combine MATCH and CREATE
+    Given any graph
+    And having executed:
+      """
+      CREATE (), ()
+      """
+    When executing query:
+      """
+      MATCH ()
+      CREATE ()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 2 |
+
+  Scenario: Combine MATCH, WITH and CREATE
+    Given any graph
+    And having executed:
+      """
+      CREATE (), ()
+      """
+    When executing query:
+      """
+      MATCH ()
+      CREATE ()
+      WITH *
+      MATCH ()
+      CREATE ()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 10 |
+
+  Scenario: Newly-created nodes not visible to preceding MATCH
+    Given any graph
+    And having executed:
+      """
+      CREATE ()
+      """
+    When executing query:
+      """
+      MATCH ()
+      CREATE ()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 1 |
+
+  Scenario: Create a single node with properties
+    Given any graph
```

Comment:
I don't think this is true. In the case of a populated uniqueness constraint, the query will not behave as specified here. I think we need `Given empty graph` here.